tests/integration/components/power-select-typeahead-test.js

```diff
@@ -16,12 +16,12 @@ test('It can select options when options are strings', function(assert) {
       {{number}}
     {{/power-select-typeahead}}
   `);
-  assert.equal($('.ember-power-select-dropdown').length, 0, 'The component is closed');
+  assert.equal(findAll('.ember-power-select-dropdown').length, 0, 'The component is closed');
```

Comment:
This is silly thing, but one of the reasons why classic `find` was divided into `find` and `findAll` is because it allows a bit more expressiveness in tests.

Doing `assert.equal(findAll('.selector').length, 1);` it's not clear if you try to assert that there is an element that may or may not be, that in this case it is, or it is an element of which it might be 0..N items, and you are asserting that there is one and only one.

When it's the former, I prefer `assert.ok(find('.selector'))`. When it's the latter I prefer `assert.equal(findAll('.selector').length, 1)` because it better simbolises that it could be two or three, while the first one is more like a true/false check.